en/organization_management/daily_management.md

```diff
@@ -0,0 +1,61 @@
+# Daily Management
+
+**NOTE**: The following operations can only be performed by organization administrators.
+
+After organization administrator login to Seafile, he/she can go to organization admin page by click **Organization Admin**
+
+![](../images/auto-upload/WX20230615-180101@2x.png)
+
+## View System Info
+
+In organization admin page, administrator can view the system usage, graphical statistics of this organization.
+
+![](../images/auto-upload/WX20230616-102525@2x.png)
+![](../images/auto-upload/WX20230616-102743@2x.png)
+
+## View Libraries and Operation Logs
+
+Administrators can not only view all libraries in this organization, but also can view some operation logs, such as file access/update and permission log.
+
+![](../images/auto-upload/WX20230616-105147@2x.png)
+![](../images/auto-upload/WX20230616-115301@2x.png)
+
+## Manage Devices
+
+Devices in organization admin page can list all the devices connected to this organization, administrator can unlink a certain device and view the detailed device connection error message.
+
+![](../images/auto-upload/WX20230616-102935@2x.png)
+![](../images/auto-upload/WX20230616-110201@2x.png)
+
+## Manage Users and Groups
+
+Administrators can import/create/delete new users, activate/deactivate users, reset users' login passwords, and set/unset normal users as administrators.
+
+![](../images/auto-upload/WX20230616-110340@2x.png)
+
+Click the user name, the administrator will enter the management interface of a single user, where the administrator can view and update some information of this user.
+
+![](../images/auto-upload/WX20230616-111042@2x.png)
+
+Like the user management interface, administrators can also view and manage groups.
+
+![](../images/auto-upload/WX20230616-111308@2x.png)
+![](../images/auto-upload/WX20230616-111327@2x.png)
+
+## Create Organizational Structure
+
+In **Departments** page, administrator can customize the organizational structure for this organization.
```

Comment:
customize -> manage

You can create departments and sub-departments. Each department can own some libraries. You can also edit member list for the departments.